packages/calypso-e2e/src/lib/components/editor-settings-sidebar-component.ts

```diff
@@ -13,25 +19,23 @@ interface Schedule {
 	meridian: 'am' | 'pm';
 }
 
-const sidebarParentSelector = '[aria-label="Editor settings"]';
+const panel = '[aria-label="Editor settings"]';
 
 const selectors = {
+	// Close button for mobile
+	mobileCloseSidebarButton: `${ panel } [aria-label="Close settings"]:visible`,
+
 	// Tab
-	tabButton: ( tabName: EditorSidebarTab ) =>
-		`${ sidebarParentSelector } button:has-text("${ tabName }")`,
+	tabButton: ( tabName: EditorSidebarTab ) => `${ panel } button[data-label="${ tabName }"]`,
 	activeTabButton: ( tabName: EditorSidebarTab ) =>
-		`${ sidebarParentSelector } button.is-active:has-text("${ tabName }")`,
+		`${ panel } button.is-active:has-text("${ tabName }")`,
 
-	// Sections
-	sectionToggle: ( sectionName: EditorSidebarSection ) =>
-		`${ sidebarParentSelector } .components-panel__body-toggle:has-text("${ sectionName }")`,
-	expandedSection: ( sectionName: EditorSidebarSection ) =>
-		`${ sidebarParentSelector } .is-opened .components-panel__body-toggle:has-text("${ sectionName }")`,
-	revisionsToggle: `${ sidebarParentSelector } .components-panel__body:has-text("Revisions")`,
-	lastSection: `${ sidebarParentSelector } .components-panel__body >> nth=-1`,
+	// General section-related
+	section: ( name: ArticleSections ) => `${ panel } button:has-text("${ name }")`,
```

Comment:
Is only looking for buttons with text possibly too generic? Like, I'm thinking of the case of "Featured image", where both the section name and the button within there has that text in there. I think we might want to use something here to specify these to be just the section headers!

Reply:
Sure, let's try narrowing this a bit more.